### ISSUE TITLE
Unphased evaluation?

### ISSUE THREAD

**Issue by rlorigro** [TimD1/vcfdist#25]
Hi, thank you for the lovely software and informative publication. Do you still have plans to extend vcfdist to evaluate unphased VCFs, as you indicated in your paper? As part of my work and I'm sure for many other groups, the majority of reference-based SV calling is unphased and in tandem repeats and so it would be great to have an unphased evaluation method that handles the representation issues you mentioned.

Thanks

**Comment by TimD1** [comment]
Yes, I am currently working on on extending vcfdist to work with unphased VCFs. Doing so requires significant changes to the underlying algorithm, and I will be working on this for the next month or two.

**Comment by rlorigro** [comment]
Awesome, thank you. looking forward to it.

**Comment by rlorigro** [comment]
That's a good point, I think it is safe to assume that we can use phased VCFs as truth
